src/gdal.mk

```diff
@@ -87,4 +87,10 @@ define $(PKG)_BUILD
     $(MAKE) -C '$(1)/ogr'   -j '$(JOBS)' install OGR_ENABLED=
     $(MAKE) -C '$(1)/apps'  -j '$(JOBS)' install
     ln -sf '$(PREFIX)/$(TARGET)/bin/gdal-config' '$(PREFIX)/bin/$(TARGET)-gdal-config'
+
+    mkdir -p '$(PREFIX)/$(TARGET)/bin
```

Comment:
I see, `usr/<target>/bin/` is empty and `git` doesn't add it. We should just create the basic layout in `mxe-conf` and put dummy files in there.